packages/common/client-utils/README.md

```diff
@@ -0,0 +1,69 @@
+# @fluid-internal/client-utils
+
+This package is intended for sharing and promoting utility functions across packages in the Fluid Framework repo.
```

Comment:
?
```suggestion
This package is intended for sharing and promoting utility functions across packages in the Fluid Framework repo, specifically the client release group.
```

Reply:
FWIW, 'core-utils' is also tied to the client release group.  I think the difference is that 'core-utils' has a policy of zero dependencies, whereas 'client-utils' has dependencies on lodash, buffer, etc.

@tylerbutler - Is that correct?

Reply:
That's correct. One is zero deps, the other is not.